mcm/reader.py

```diff
@@ -233,7 +232,7 @@ def num_columns(self):
 
     def headers(self):
         """original ordered list of spreadsheet headers"""
-        return self.csvreader.fieldnames
+        return self.csvreader.unicode_fieldnames
```

Comment:
Also need to consider if the bad headers come from a xls file and the `headers` method from ExcelReader